src/mame/drivers/atetris.cpp

```diff
@@ -163,6 +163,23 @@ void atetris_state::coincount_w(uint8_t data)
 }
 
 
+void atetris_state::atetrisbp_w(uint8_t data)
```

Comment:
Naming of this handler function is poor.
Please consider subclassing `atetris_state` into something like `atetris_bartop_state` and move & rename this function as a private one to something more meaningful as `output_w` or w/e.

Reply:
Good idea, I'll make that change shortly.